README.md

```diff
@@ -990,6 +990,12 @@ IEEE Trans. Image Process. (TIP), 33:1826-1837, 2024. [PDF](https://arxiv.org/pd
 - <a name="Zhang24tip"></a>Zhang, P., Liu, H., Ge, Z., Wang, C., Lam, E. Y.,   
 *[Neuromorphic Imaging with Joint Image Deblurring and Event Denoising](https://doi.org/10.1109/TIP.2024.3374074)*,  
 IEEE Trans. Image Process. (TIP), 2024. [PDF](https://arxiv.org/pdf/2309.16106.pdf), [Code](https://github.com/pz-even/ni-joint)
+- <a name="Muglikar23arxiv"></a>Zou.R, Muglikar.M, Messikommer.N, Scaramuzza. D,   
+*[Seeing behind Dynamic Occlusions with Event Cameras](https://arxiv.org/abs/2307.15829)*,  
+Arxiv, 2024. [PDF](https://arxiv.org/abs/2307.15829)
```

Comment:
No need for this PDF link if there is no official doi link used in the title of the paper.
I would use this PDF link on the title of the paper until a better doi is obtained (when officially published).